DZ_5/Task_28.py

```diff
@@ -0,0 +1,22 @@
+# Задача 28: Напишите рекурсивную функцию sum(a, b),
+# возвращающую сумму двух целых неотрицательных чисел.
+# Из всех арифметических операций допускаются только +1 и -1.
+# Также нельзя использовать циклы.
+#
+# *Пример:*
+#
+# 2 2
+#     4
+
+
+a = int(input("Введите А: "))
+b = int(input("Введите В: "))
+
+def sum(a, b):
+    if a == 0:
+        print(b)
+        return b
+    return sum(a-1, b+1)
+
+sum(a,b)
```

Comment:
Почему вы используете `print` именно так, когда логичней вот так:
```suggestion
def sum(a, b):
    if a == 0:
        return b
    return sum(a - 1, b + 1)


print(sum(a, b))
```
Объясните

Reply:
Да на самом деле и объяснить не смогу. Пока учусь пробую писать как мне, кажется, правильно. Спасибо за замечания, буду брать на заметку.